newIDE/app/src/AssetStore/index.js

```diff
@@ -68,8 +71,11 @@ export const AssetStore = ({ project }: Props) => {
     filtersState,
   } = navigationState.getCurrentPage();
   const searchBar = React.useRef<?SearchBarInterface>(null);
+
   const shouldAutofocusSearchbar = useShouldAutofocusSearchbar();
-  const [isFiltersPanelOpen, setIsFiltersPanelOpen] = React.useState(false);
+  const [isFiltersPanelOpen, setIsFiltersPanelOpen] = React.useState(
+    !isOnHomePage && !openedAssetShortHeader
```

Comment:
This means the panel is opened when going back from details of an asset to the result list?

Reply:
I think this state is initialized only when the dialog is opened back.